### DISCUSSION TITLE
Fix Quét mã Momo does not yet support this block. This may impact the shopper experience

### DISCUSSION THREAD

**Discussion by Vo-Thi-Thu-Suong** [woocommerce/woocommerce#45869] [Checkout Flow]
[
<img width="187" alt="Screenshot 2024-03-24 211910" src="https://github.com/woocommerce/woocommerce/assets/164763172/ecac4b0b-192e-416f-8db9-9df1fb1d7d59">
](url)

**Comment by ralucaStan** [comment]
Hello @Vo-Thi-Thu-Suong, thank you for highlighting this incompatibility. 

Currently, it's up to extension developers to add support for the new checkout flow offered via the Cart and Checkout Blocks. I recommend getting in touch with them. 

We have some [documentation for Third-party developers](https://github.com/woocommerce/woocommerce/tree/4f2ed8d2627ab53f997b1f1d937f2520e1a98115/plugins/woocommerce-blocks/docs#third-party-developers) and some [Developer Resources](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce-blocks/docs#developer-resources) available to help with their integration efforts.
